src/CoreBundle/Security/BackendPageAccessCheckInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\Security;
+
+use ChameleonSystem\CoreBundle\DataModel\CmsMasterPagdef;
+
+interface BackendPageAccessCheckInterface
+{
+    /**
+     * Checks for page access of the current user/client to the current page.
+     * In case of failure this redirects to the login page.
+     *
+     * @return void
+     */
+    public function assertAccess();
```

Comment:
I would remove assertAccess. I think things like redirection belong into the controller, not into this service.

Reply:
(it was present and called like this before in ChameleonBackendController::getResponse - only missing in the interface)

Reply:
It is a reason to do it (only) for consistency reasons.

At least the typing in ChameleonBackendController::getResponse() was wrong before.

More was not changed here.